napari/_qt/qt_viewer.py

```diff
@@ -202,8 +219,24 @@ def _update_palette(self, palette):
 
     def on_mouse_move(self, event):
         """Called whenever mouse moves over canvas.
+        when axis is not None and the camera is turntable, on moving the mouse
+        one can rotate around the vertical axes.
         """
         layer = self.viewer.active_layer
+        if (
+            self.axis is not None
+            and self.view.camera.name == "TurntableCamera"
+        ):
+            self.axis.transform.reset()
+
+            self.axis.transform.rotate(self.view.camera.roll, (0, 0, 1))
+            self.axis.transform.rotate(self.view.camera.elevation, (1, 0, 0))
+            self.axis.transform.rotate(self.view.camera.azimuth, (0, 1, 0))
+
+            self.axis.transform.scale((50, 50, 0.001))
```

Comment:
what are these `50` numbers?

Reply:
the length of x, y, z axis that is displayed in green. Turntable camera only rotates in z hence z is scaled but x, y remain same as what it was initialized with